test/Tester/CancellationTests/CancellationTokenPassingTests.cs

```diff
@@ -0,0 +1,122 @@
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Orleans;
+using Orleans.TestingHost;
+using UnitTests.GrainInterfaces;
+using Xunit;
+using UnitTests.Tester;
+
+namespace UnitTests.MembershipTests
+{
+    public class CancellationTokenPassingTests : HostedTestClusterPerTest
```

Comment:
"CancellationTokenTests" seems like testing the cancellation token itself. But if it's just me - I can change to this name.